content/about/_index.md

```diff
@@ -29,35 +29,33 @@ serve all Computer Science students.
 **You don't have to! As long as you're a Computer Science student, you're a
 member!**
 
-### What we can do for you
-
-Expect BBQs (weather permitting), free breakfast, board games nights, the Career
-Fair, Industry Panels and Info Sessions, volunteer opportunities within the
-Computer Science Department, organizing the highly anticipated Career Fair, and
-holding social events such as licensed parties and the year-end boat cruise!
-
-We also distribute various free stuff, including annual dayplanners with monthly
-view, events/volunteer opportunities available through CSSS and the CS
-Department, a central map of UBC at the back; glossy CSSS pins; and lanyards,
-which are a beautiful blue colour and have a lobster clip attached at the end.
-
-So come by and socialize with your CS peers: hang out, eat and be merry! We will
-organize many events just for CS students. We can also help you connect with the
-CS Department: we can't stress how important it is to dip your feet into
-volunteer opportunities within the CS department. You will really find a home in
-CS by helping out within the department. Increase your connections by networking
-with industry members at the career fair. Of course, we also have cheap food
-([Food Price List](/cube/menu)). Forget those greedy vending machines. We're not
+### What can we do for you?
+
+Expect BBQs (weather permitting), free breakfasts, board games nights, 
+Industry Panels and Info Sessions, volunteer opportunities within the Computer Science department, 
+the highly-anticipated Career Fair, and social events such as our year-end boat cruise!
```

Comment:
We should probably change this to "gala", despite the old copy talking about the cruise.

Reply:
Agree